pattern-2/bosh-release/jobs/health_check/spec

```diff
@@ -0,0 +1,9 @@
+---
+name: health_check
+
+templates:
+    run.sh: bin/run
+
+packages: []
+
+properties: {}
```

Comment:
Shall we add an extra empty line at the end of the file?

Reply:
fixed with https://github.com/wso2/pivotal-cf-apim/pull/49/commits/328ee6d6be3730567554d39a86a2c2829dd01c5c